### ISSUE TITLE
Klicking on a room-icon doesen't work.

### ISSUE THREAD

**Issue by ChristophKuhn** [pdaddyo/soundbounce#149]
If i click on the icon of a room (for example "Liquifaction"), The caption of the room icon says: Joining... After a socond it turns back to "Liquifaction" and nothing happens. 

Spotify is Ok here (at work) und I have a Premium Spotify Account.

Besides: I'm at work. At Home everything is OK.

**Comment by pdaddyo** [comment]
This means websockets are blocked at work, I'm afraid.  We use port 80,
please check this list and let us know which ports are open for you at
work.  Thanks!

http://websocketstest.com/

**Comment by ChristophKuhn** [comment]
So I Can't use Sondbounce?  That's really bad :-( !
Is there no way I can use soundbounce at work? I used Sondrop a long time ! Eberything worked fine with it at work.

WebSockets seem to
Not Work for you :(
Environment
WebSockets supported	Yes✔
HTTP Proxy	YES 
1.1 proxy005.stadt.hamburg.de:3128 (squid/2.7.STABLE9)
WebSockets (Port 80)
Connected	No✘ Disconnected at 9:9:48
WebSockets (Port 443)
Connected	No✘ Disconnected at 9:9:48
WebSockets (Port 8080)
Connected	No✘ Disconnected at 9:9:48
WebSockets (Port 443, SSL)
Connected	No✘ Disconnected at 9:9:48

**Comment by ChristophKuhn** [comment]
This thread is not closed.

**Comment by pdaddyo** [comment]
Sorry Christoph, it's closed because we're not planning on adding other communication methods, so will mark as wontfix.

**Comment by fertilelies** [comment]
How about the ability to use other Ports other than 80? I have WebSockets, but it looks like port 80 was recently blocked. If there were some "Advanced Options" or something where I could set the port to one that I know is open, if possible, that would be really cool.

**Comment by pdaddyo** [comment]
We're looking into adding exactly this on the login screen - advanced options to allow connections on 443 / 8080.  Follow us on facebook and I'll announce once this is in place.

**Comment by pdaddyo** [comment]
@fertilelies This is now live!  You can now choose between ports 80, 443 and 8080 on login screen.

**Comment by fertilelies** [comment]
Works as advertised, you are awesome!

**Comment by pdaddyo** [comment]
Thanks, glad to hear it's working!

**Comment by ChristophKuhn** [comment]
The new Feature worked for me! If I chosse Port 443 on startup, everything is fine! First I didn't hear any music first, but after i deleted the cache folder it works for me ! I'm now a really great soundbounce fan ! We have to manage to get more people in, now !!!
